demo/index.html

```diff
@@ -19,191 +19,202 @@
+        <paper-dropdown-menu label="Menu tabs!?">
+          <paper-tabs class="dropdown-content">
+            <paper-tab>cheddar</paper-tab>
+            <paper-tab>stilton</paper-tab>
+            <paper-tab>emmental</paper-tab>
+          </paper-tabs>
+        </paper-dropdown-menu>
+      </template>
+    </demo-snippet>
+
+    <h4>You can remove the ripple and the animations</h4>
+    <demo-snippet class="centered-demo">
+      <template>
+        <paper-dropdown-menu label="Dinosaurs" noink no-animations>
+          <paper-listbox class="dropdown-content">
+            <paper-item>allosaurus</paper-item>
+            <paper-item>brontosaurus</paper-item>
+            <paper-item>carcharodontosaurus</paper-item>
+            <paper-item>diplodocus</paper-item>
+          </paper-listbox>
+        </paper-dropdown-menu>
+      </template>
+    </demo-snippet>
+
+    <h4>You can style a paper-dropdown-menu using custom properties</h4>
+    <demo-snippet class="centered-demo">
+      <template>
+        <style is="custom-style">
+          paper-dropdown-menu.custom {
+            --paper-input-container-label: {
+              color: white;
+              background-color: var(--paper-pink-500);
+              font-style: italic;
+              text-align: center;
+              font-weight: bold;
+            };
+            --paper-input-container-input: {
+              color: var(--paper-indigo-500);
+              font-style: normal;
+              font-family: serif;
+              text-transform: uppercase;
+            }
+            /* no underline */
+            --paper-input-container-underline: {
+              display: none;
+            };
+          }
+        </style>
+        <paper-dropdown-menu class="custom" label="Custom" no-label-float>
+          <paper-listbox class="dropdown-content">
+            <paper-item>allosaurus</paper-item>
+            <paper-item>brontosaurus</paper-item>
+            <paper-item>carcharodontosaurus</paper-item>
+            <paper-item>diplodocus</paper-item>
+          </paper-listbox>
+        </paper-dropdown-menu>
+      </template>
+    </demo-snippet>
+  </div>
 
   <script>
-    Demo.letters = [
-      'alpha',
-      'beta',
-      'gamma',
-      'delta',
-      'epsilon'
-    ];
+  document.addEventListener("WebComponentsReady", function() {
     Demo.dinosaurs = [
-      'allosaurus',
-      'brontosaurus',
-      'carcharodontosaurus',
-      'diplodocus',
-      'ekrixinatosaurus',
-      'fukuiraptor',
-      'gallimimus',
-      'hadrosaurus',
-      'iguanodon',
-      'jainosaurus',
-      'kritosaurus',
-      'liaoceratops',
-      'megalosaurus',
-      'nemegtosaurus',
-      'ornithomimus',
-      'protoceratops',
-      'quetecsaurus',
-      'rajasaurus',
-      'stegosaurus',
-      'triceratops',
-      'utahraptor',
-      'vulcanodon',
-      'wannanosaurus',
-      'xenoceratops',
-      'yandusaurus',
-      'zephyrosaurus'
+      "allosaurus",
```

Comment:
Why did you change all of these quotes? I thought we prefer `'` over `"` in JS..